src/components/molecules/NavSearchBar/NavSearchBar.tsx

```diff
@@ -147,7 +147,7 @@ const NavSearchBar = () => {
   const clearSearchIcon = (
     <img
       className="NavSearchBar__clear-search"
-      src={CloseIcon}
+      src={navDropdownCloseIcon}
       alt="close button"
       onClick={clearSearchQuery}
     />
```

Comment:
Wouldn't it be nicer to remove it & use font awesome instead?

Reply:
Potentially, though @mcflyok tends not to like the font awesome iconography in general I believe.

This was just renaming the variable name of existing code, not intending to change logic in this PR.